jujugui/static/gui/src/app/components/status/status.js

```diff
@@ -109,6 +109,59 @@ class Status extends React.Component {
   }
 
   /**
+    Navigate to the chosen application.
+    @param appId {String} The id of the application to display.
```

Comment:
Yes but the id is the name AFAIK, we don't have another separate id, which this could suggest.